inst/Examples/2ArmTimeToEventOutcomeAnalysis/Description.Rmd

```diff
@@ -1,29 +1,51 @@
+
+The examples included here are to provide different approaches for analyzing the data in the trial. 
+
+In the RCode directory of this example you will find the following R files:
+
+1. AnalyzeUsingEastLogrankFormula.R - Contains a function named AnalyzeUsingEastLogrankFormula to demonstrate the R code necessary for this example as described below.
+2. AnalyzeUsingSurvivalPackage.R - Contains a function named AnalyzeUsingSurvivalPackage to demonstrate the R code necessary for this example as described below.
+3. AnalyzeUsingHazardRatioLimitsOfCI.R - Contains a function named AnalyzeUsingHazardRatioLimitsOfCI to demonstrate the R code necessary for this example as described below.
+
+
+## Example 1 - Two Arm TTE Analysis Using a Formula 28.2 in the East manual
+
+To replace the analysis, use the formulas Q.242 and Q.243 in the East manual to compute the statistic.  The purpose of this example is to demonstrate how the analysis and decision making could be modified in a simple approach.  The test statistic is compared to the lower boundary computed and sent by East as an input. This example does NOT include a futility rule. 
+
+## Example 2 - Two Arm TTE Analysis Using survdiff function from the survival package
+
+This example utilizes the survdiff function from survival package to perform the analysis. The chisquare statistic from survdiff is used to compute the logrank statistic that is compared to the lower boundary computed and sent by East as an input.  This example does NOT include a futility rule.
+
+## Example 3 - Utilization of Confidence Interval Limits for Go/No Go Decision Making in two arm Survival analysis
+
+In many phase II trials, teams like to utilize the upper and lower boundaries of a confidence interval (CI) for making Go and No Go type decision. In this type of trial, teams often set a Minimum Acceptable Value (MAV), the minimum treatment difference that would warrant further development in a product, and a Target Value (TV), the value that is highly desirable based on other consideration.  In this simplified example, if it is likely that the Hazard ratio is below the MAV then a Go decision is made. If a Go decision is not made, then if is is unlikely that the Hazard Ratio is below the TV, a No Go decision is made.      
+
+In this example, the coxph() from "survival" package in R is utilized to analyze the data and compute at a user-specified confidence interval. The team would like to make a Go decision if there is at least a 90% chance that the Hazard Ratio is less than the MAV.  If a Go decision is not made, then a No Go decision is made if there is less than a 10% chance the Hazard ratio is less than the TV.  Using a frequentist CI an approximation to this design can be done by the logic described below.
+
+#At an Interim Analysis
+    If the Lower Limit of the CI, denoted by LL, is less than a log of user-specified amount then a Go decision is made.  Specifically, if LL < log user-specified lower limit --> Go

```

Comment:
This does not match what is in the code.  The code uses MAV, TV.   Please update this to match and also update the rule per comment above.

Reply:
Updated